src/OrchardCore/OrchardCore.OpenId.Core/YesSql/Models/OpenIdApplication.cs

```diff
@@ -25,20 +24,23 @@ public class OpenIdApplication
         public string ClientSecret { get; set; }
 
         /// <summary>
-        /// Gets or sets the consent type
-        /// associated with the current application.
+        /// Gets or sets the consent type associated with the current application.
         /// </summary>
         public string ConsentType { get; set; }
 
         /// <summary>
-        /// Gets or sets the display name
-        /// associated with the current application.
+        /// Gets or sets the display name associated with the current application.
         /// </summary>
         public string DisplayName { get; set; }
 
         /// <summary>
-        /// Gets or sets the physical identifier
-        /// associated with the current application.
+        /// Gets or sets the localized display names associated with the application.
+        /// </summary>
+        public ImmutableDictionary<CultureInfo, string> DisplayNames { get; set; }
```

Comment:
FYI, I tested manually and it works just fine (most likely because JSON.NET supports `TypeConverter`, for which there's a dedicated `CultureInfoConverter` to convert `CultureInfo` instances from/to strings):

```json
{
    "DisplayNames":{"fr-FR":"Ma super app"}
}
```